client/src/components/page/Sidebar/index.tsx

```diff
@@ -0,0 +1,69 @@
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { RouteComponentProps, withRouter } from 'react-router';
+import './Sidebar.css';
+import { DashboardOutlined } from '@ant-design/icons';
+import { Dictionary } from '@onaio/utils';
+import { Layout, Menu } from 'antd';
+import Logo from '../../../assets/images/opensrp-logo-color.png';
+import { Link } from 'react-router-dom';
+import { URL_HOME } from '../../../constants';
+
+/** interface for SidebarProps */
+export interface SidebarProps extends RouteComponentProps {
+  authenticated: boolean;
+  extraData: { [key: string]: Dictionary };
+}
+
+/** default props for Sidebar */
+const defaultSidebarProps: Partial<SidebarProps> = {
+  authenticated: false,
+};
+
+/** The Sidebar component */
+
+export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) => {
+  const { extraData } = props;
+  const { roles } = extraData;
+
+  return (
+    <Layout.Sider width="275px">
+      <div className="logo">
+        <Link to={URL_HOME}>
+          <img src={Logo} className="img-fluid" alt="" />
+        </Link>
+      </div>
+      <Menu theme="dark" defaultSelectedKeys={['1']} mode="inline">
+        <Menu.SubMenu key="admin" icon={<DashboardOutlined />} title="Admin">
+          {roles && roles.includes('ROLE_EDIT_KEYCLOAK_USERS') && (
+            <Menu.Item key="users">
+              <Link to="/admin" className="admin-link">
+                Users
+              </Link>
+            </Menu.Item>
+          )}
+          <Menu.Item key="teams">
+            <Link to="/teams" className="admin-link">
```

Comment:
These should ideally be defined in constants, i.e. the url locations

Reply:
Ok i am updating now